.github/workflows/rucio-release-images.yml

```diff
@@ -0,0 +1,41 @@
+name: Rucio Release Image CI
+
+on:
+  push:
+    tags:
+      - "release-*.cms*"
+
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image: [rucio-probes, rucio-daemons, rucio-server]
+      fail-fast: True
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Get git tag name
+        id: gittag
+        run: echo "tag=${GITHUB_REF#refs/tags/}" >> "$GITHUB_ENV"
+
+      - name: Get rucio version
+        id: rucioversion
+        run: echo "rucio_version=$( echo ${{ env.tag }} |  grep -Eo '[0-9]+\.[0-9]+\.[0-9]')" >> "$GITHUB_ENV"
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
```

Comment:
Do they have podman for all of this? That's what I've been using instead of docker since docker (the company) is becoming more hostile to open projects.

Reply:
Ok, will use `podman buildah` instead. 
I have heard of docker removing open source projects from the hub, but not open use of the cli would be too dangerous for them.

Reply:
Done